packages/client/wallets/aa/src/adapters/reservoir.ts

```diff
@@ -1,22 +1,22 @@
 import { ReservoirWallet } from "@reservoir0x/reservoir-sdk";
 import { hexToBigInt, http } from "viem";
 
-import { getBundlerRPC } from "../blockchain/BlockchainNetworks";
+import { getBundlerRPC } from "..";
 import { EVMAAWallet } from "../blockchain/wallets/EVMAAWallet";
 
-export function reservoirAdapter(aaWallet: EVMAAWallet): ReservoirWallet {
+export function reservoirAdapter(wallet: EVMAAWallet): ReservoirWallet {
     return {
-        address: async () => aaWallet.getAddress() as string,
+        address: async () => wallet.getAddress(),
```

Comment:
Since `ReservoirWallet.address` expects `() => Promise<string>` it'll cast this for us because `0x${string}` is compatible